crowdin_content.yaml

```diff
@@ -1,19 +1,42 @@
 # Project settings
 project_identifier: "cs-unplugged"
 api_key_env: "CROWDIN_API_KEY"
-base_path: ""
 
 # Preserve directory tree during upload
 preserve_hierarchy: true
 
 files: [
     {
         source: "/csunplugged/topics/content/en/**/*.yaml",
-        translation: "/csunplugged/topics/content/%two_letters_code%/**/%original_file_name%",
-
+        translation: "/csunplugged/topics/content/%osx_locale%/**/%original_file_name%",
+        languages_mapping: {
+            osx_locale: {
+                "en-UD": "xx_LR",  # Pseudo language
```

Comment:
Why is "xx_RL" not listed here?

Reply:
xx_RL is on the CSU end only, it's not a language crowin knows anything about (and is symlinked to xx_LR)